app/javascript/src/component_dialog_validation.js

```diff
@@ -74,8 +74,10 @@ class DialogValidation {
     if(this.#config.activator) {
       this.#config.activator.focus();
     }
-    this.$node.dialog("close");
-    this.$node.dialog('destroy'); 
+    if(this.$node.dialog("instance")) {
```

Comment:
This is a quick fix.  It works fine, and a more robust solution is already included within my reafctored dialog branch, which can hopefully be merged soon.